index.js

```diff
@@ -31,7 +31,7 @@ class ghostGoogleDrive extends StorageBase {
    */
   save(file, targetDir) {
     const _this = this;
-    return new Promise(function(resolve, reject) {
+    return new Promise(function (resolve, reject) {
```

Comment:
Consider undoing this change. It adds "noise" to the PR and distracts from meaningful changes elsewhere.

Reply:
done